docker/install-deps

```diff
@@ -223,6 +223,7 @@ fi
 if [ ! -f /usr/local/lib/libeatmydata.so ]; then
     git clone https://github.com/stewartsmith/libeatmydata /tmp/libeatmydata
     pushd /tmp/libeatmydata
+        git checkout df7ddeb0345104f25b5b7bb154bc6a008c8c8404
```

Comment:
please add comment why do we need exactly this commit

Reply:
<img width="1318" alt="Screenshot 2021-04-14 at 20 53 14" src="https://user-images.githubusercontent.com/13628059/114756474-7084b300-9d63-11eb-9153-7d7ea0924ef6.png">
This is the last working commit on CentOS 6